package.json

```diff
@@ -38,47 +38,47 @@
   "peerDependencies": {
     "@angular/core": ">=13.0.0",
     "@messageformat/core": "^3.2.0",
-    "@ngx-translate/core": "^14.0.0 || ^15.0.0"
+    "@ngx-translate/core": "^14.0.0 || ^15.0.0 || ^18.0.0"
```

Comment:
v18 is Angular, this should be v16. :slightly_smiling_face:

Reply:
yep
fixed